src/applications/appeals/shared/utils/useBrowserMonitoring.js

```diff
@@ -43,6 +45,37 @@ const initializeRealUserMonitoring = customRumSettings => {
   }
 };
 
+// https://docs.datadoghq.com/logs/log_collection/javascript/?tab=us#configuration
+const defaultLogSettings = {
+  // Custom settings
+  // applicationId: '{UUID}',
+  // clientToken: 'pubb{ID}',
+  // service: 'benefits-{app name}',
+  // version: '1.0.0',
+
+  // default settings
+  site: 'ddog-gov.com',
+  // see src/site/constants/vsp-environments.js for defaults
+  env: environment.vspEnvironment(), // 'production'
+  sessionSampleRate: 100,
+  forwardErrorsToLogs: true,
+  forwardConsoleLogs: ['error'],
+  forwardReports: [],
+  telemetrySampleRate: 20, // default
```

Comment:
Is telemetry the thing that provides the error logs we're looking for? And if so, is there any reason why we can't bump it up to 100?

Reply:
Looked this up with @Mottie - unclear from docs whether this applies to error logs from our app, or DD's SDK only. Will merge as-is, and just see what happens in staging.